src/writeFile.js

```diff
@@ -85,7 +85,9 @@ export default function writeFile(globalRef, pattern, file) {
                 // A hack because loaderUtils.interpolateName doesn't
                 // find the right path if no directory is defined
                 // ie. [path] applied to 'file.txt' would return 'file'
+                let slashAdded = false;
```

Comment:
I moved it down here so it's consistent with the `dotRemoved` block. So now your guys' code comments also serve as comments for the existence of the newly introduced variable. Hope this will suffice.

Reply:
`slashAdded` => `hasSlash`